packaging/debian/control

```diff
@@ -0,0 +1,38 @@
+Source: ansible-runner
+Section: admin
+Priority: optional
+Maintainer: Ansible <info@ansible.com>
+Build-Depends: debhelper (>= 11), dh-python, python-all, python-setuptools, python3-all, python3-setuptools, python-pytest, python3-pytest, python-pexpect (>= 4.5), python3-pexpect (>= 4.5), python-mock, python3-mock, python-psutil, python3-psutil, python3-yaml
+Standards-Version: 4.1.3
+Homepage: https://www.ansible.com
+X-Python-Version: >= 2.7
+X-Python3-Version: >= 3.5
+Vcs-Git: https://github.com/ansible/ansible-runner.git
+
+Package: python-ansible-runner
```

Comment:
Will users be able to `apt-get install ansible-runner` and get the correct version for whatever Python they have installed?  Or will they have to `apt-get install python-ansible-runner`?  Would aesthetically prefer the former.

Reply:
I'd also prefer this to be just `ansible-runner`, to match our RPM packaging. Runner is a more than just a python module, which is what usually follows this `python-` naming convention.

Reply:
@ghjm @shanemcd the most straightforward way of doing so is to ignore Python 3 support.  Since 2.7 is still the default version for Debian 9 and both Ubuntu LTS versions we're talking about, I suppose that's not too big a deal.

Reply:
For the upcoming changes to the RPM, I have created a metapackage called `ansible-runner`, that `Requires: python2-ansible-runner` on on EL7, and `Requires: python3-ansible-runner` on platforms where that is the default. I'm not sure if something like this would make sense for Debian.

Reply:
*both* Ubuntu LTS?

Reply:
@wenottingham 16.04 and 18.04 were specified.  14.04 is End of Life as of next month.

Reply:
@shanemcd Debian/Ubuntu has metapackages, but none of these targets are Python 3-first so creating one in this case seems pointless.  Additionally, it's easy on Debian-based systems to have multiple versions of Python installed at the same time, so I had thought that it made sense (and was certainly the direction that the build tools pushed me in) to have explicit per-Python packages like this so that the user can decide which makes the most sense for them.

With all that said, how strongly do we still feel that there should be an `ansible-runner` metapackage that only points to the Python 2 package?  cc @ghjm